public/search.js

```diff
@@ -97,6 +103,19 @@ function displayAxis(times)
   }
 }
 
+var getDocColl = function(uri) {
+  $.ajax({
+    url: '/v1/documents?uri='+uri+'&category=collections&format=json',
+    success: function(data, textStatus) {
+      console.log('got collections: ' + data);
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      console.log('problem');
+    },
+    async: false,
+  });
+}
+
```

Comment:
I was trying to use the function in the other js file but I couldn't access it. I was having trouble trying to include the search.js file in the index page, so I need help with this